src/Components/ProvisioningWizard/steps/ReservationProgress/index.js

```diff
@@ -124,54 +125,56 @@ const ReservationProgress = ({ setLaunchSuccess }) => {
-                return (
-                  <ProgressStep
-                    variant={variant}
-                    id={name}
-                    key={name}
-                    icon={step > currentStep ? <PendingIcon /> : undefined}
-                    isCurrent={step === currentStep}
-                    description={description}
-                    titleId={name}
-                    aria-label={`step ${name} ${variant}`}
-                    popoverRender={
-                      currentError && step === currentStep
-                        ? (stepRef) => (
-                            <Popover
-                              aria-label={`${name} error message`}
-                              headerContent={<div>Error</div>}
-                              bodyContent={<div>{currentError}</div>}
-                              reference={stepRef}
-                              position="right"
-                            />
-                          )
-                        : undefined
-                    }
-                  >
-                    {name}
-                  </ProgressStep>
-                );
-              })}
-            </ProgressStepper>
+            {!polledReservation?.success && (
+              <ProgressStepper isCenterAligned>
+                {steps.map(({ name, description }, step) => {
+                  const variant = mapCurrentVariant(step, currentStep, currentError);
+                  return (
+                    <ProgressStep
+                      variant={variant}
+                      id={name}
+                      key={name}
+                      icon={step > currentStep ? <PendingIcon /> : undefined}
+                      isCurrent={step === currentStep}
+                      description={description}
+                      titleId={name}
+                      aria-label={`step ${name} ${variant}`}
+                      popoverRender={
+                        currentError && step === currentStep
+                          ? (stepRef) => (
+                              <Popover
+                                aria-label={`${name} error message`}
+                                headerContent={<div>Error</div>}
+                                bodyContent={<div>{currentError}</div>}
+                                reference={stepRef}
+                                position="right"
+                              />
+                            )
+                          : undefined
+                      }
+                    >
+                      {name}
+                    </ProgressStep>
+                  );
+                })}
+              </ProgressStepper>
+            )}
           </EmptyStateBody>
           {polledReservation?.success && PROVIDERS_INSTANCES_SUPPORT.includes(provider) && (
             <InstancesTable provider={provider} region={chosenRegion} reservationID={reservationID} />
           )}
           {/* TODO: remove hidden input */}
```

Comment:
Also, let's complete this TODO. You have provided ouiaId="launch-id" above, and the test should be looking for that, not the hidden reservation_id anymore,